KOF/Game/Models/Player.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace KOF.Models
+{
+    public class Player
+    {
+        public string PID { get; set; }
+        public string Email { get; set; }
+        public string DisplayName { get; set; }
+        public string PhotoID { get; set; }
+        public bool IsPaidUser { get; set; }
```

Comment:
What is the purpose of this attribute? I think is not gonna be necessary because we're not gonna include payments in the game